sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/RocksDB.scala

```diff
@@ -156,6 +156,8 @@ class RocksDB(
   private val byteArrayPair = new ByteArrayPair()
   private val commitLatencyMs = new mutable.HashMap[String, Long]()
```

Comment:
@dylanwong250 please can we reset this `commitLatencyMs` map too just like you do for `loadMetrics`. We currently don't reset it and might be returning metrics from previous batch (existing bug). Thanks.